src/pages/home/HeaderView.js

```diff
@@ -39,17 +40,13 @@ const defaultProps = {
 const HeaderView = props => (
     <View style={[styles.appContentHeader]}>
         <View style={[styles.appContentHeaderTitle]}>
-            {props.shouldShowHamburgerButton && (
-                <TouchableOpacity
-                    onPress={props.onHamburgerButtonClicked}
+            {props.shouldShowNavigationMenuButton && (
+                <Pressable
```

Comment:
I don't think this is a blocker, but I noticed `Pressable` had a `hitSlop` property we could define that would make the back button easier to hit, especially on smaller devices. I noticed on the iOS emulator that the back button hit area didn't extend all the way to the left/topmost corner.

Reply:
Cool do you know if this works on web? I started messing around with it. But it doesn't seem to do anything.

Reply:
Can definitely fix the issue you are describing. Just not sure if `hitSlop` is the way since it is not working on web.

Reply:
Wow, works great now on the emulator. You're right in that adding to the padding is more robust and I think a better solution (previously thought that we could've done something like `hitSlop={{top: 10, left: 20, bottom: 10, right: 10}}` but it didn't change much)